src/test/java/guitests/FilterTests.java

```diff
@@ -64,6 +64,91 @@ public void milestoneAliasFilterTest() {
         assertEquals(0, issuePanel.getIssueCount());
     }
 
+    @Test
+    public void countFilterTest(){
+        ListPanel issuePanel = find("#dummy/dummy_col0");
+
+        // Checking 2 issues show for count:2
+        click("#dummy/dummy_col0_filterTextField");
+        selectAll();
+        type("count:2");
+        push(KeyCode.ENTER);
+
+        assertEquals(2, issuePanel.getIssueCount());
+
+        // Checking 7 issues shown for count:7
+        click("#dummy/dummy_col0_filterTextField");
+        selectAll();
+        type("count:7");
+        push(KeyCode.ENTER);
+
+        assertEquals(7, issuePanel.getIssueCount());
```

Comment:
Just one non boundary case is enough? If 2 is broken, 7 has to be broken as well right?